OutOfSchool/OutOfSchool.DataAccess/Repository/AboutPortalRepository.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using OutOfSchool.Services.Models;
+
+namespace OutOfSchool.Services.Repository
+{
+    public class AboutPortalRepository : SensitiveEntityRepository<AboutPortal>, IAboutPortalRepository
+    {
+        private readonly OutOfSchoolDbContext db;
+
+        public AboutPortalRepository(OutOfSchoolDbContext dbContext)
+            : base(dbContext)
+        {
+            db = dbContext;
+        }
+
+        public IUnitOfWork UnitOfWork => db;
```

Comment:
Don't see if `UnitOfWork` is used anywhere. Probably remove it.

Reply:
I removed it.